openedx/core/djangoapps/user_authn/api/tests/test_views.py

```diff
@@ -159,3 +161,164 @@ def test_tpa_hint(self):
 
         response = self.client.get(self.url, self.query_params)
         assert response.data['providers'] == provider_data
+
+
+@skip_unless_lms
+@ddt.ddt
+class MFEAppContextViewTest(ThirdPartyAuthTestMixin, APITestCase):
```

Comment:
Why rewriting all the test already written in `TPAContextViewTest`? Just inherit this class from `TPAContextViewTest` and it will include all those into it.